OrbitCore/Params.h

```diff
@@ -32,6 +32,7 @@ struct Params
     bool  m_BpftraceCallstacks;
     bool  m_SystemWideScheduling;
     bool  m_UseBpftrace;
+    bool  m_SampleWithPerf;
```

Comment:
As mentioned, something like this should not be a boolean. It should be an enum stating the possible options, that is, something like:

enum {
  SAMPLE_WITH_PERF,
  SAMPLE_WITH_BPFTRACE,
  SAMPLE_WITH_PERF_EVENT_OPEN, 
};

Reply:
I'm not sure I want to refactor this when we're going to remove BpfTrace and Perf as soon as possible.